iot-hub/Samples/service/CleanupDevicesSample/CleanupDevicesSample.cs

```diff
@@ -126,7 +151,29 @@ private async Task CleanupDevices()
                         inputBlobContainerUri: storageAccountSasUri,
                         outputBlobContainerUri: storageAccountSasUri,
                         storageAuthenticationType: StorageAuthenticationType.KeyBased);
-                JobProperties importDevicesToBeDeletedJob = await _registryManager.ImportDevicesAsync(importDevicesToBeDeletedProperties);
+
+                JobProperties importDevicesToBeDeletedJob = null;
+
+                tryCount = 0;
+                while (true)
+                {
+                    try
+                    {
+                        importDevicesToBeDeletedJob = await _registryManager.ImportDevicesAsync(importDevicesToBeDeletedProperties);

```

Comment:
```suggestion
                        importDevicesToBeDeletedJob = await _registryManager
                            .ImportDevicesAsync(importDevicesToBeDeletedProperties)
                            .ConfigureAwait(false);

```

Reply:
From what I know, we don't use ConfigureAwait in samples.